config/detekt.yml

```diff
@@ -1,22 +1,3 @@
-failFast: false
-
-test-pattern: # Configure exclusions for test sources
```

Comment:
Configured in `detekt {}` task.

Reply:
Could you help me understand where this is defined? I can't seem to find where these are configured in the detekt block.